runtime/lua/vim/_editor.lua

```diff
@@ -735,6 +735,19 @@ function vim._cs_remote(rcid, server_addr, connect_error, args)
   }
 end
 
+--- Display a deprecation notification to the user.
+---
+---@param name        string     Deprecated function.
+---@param alternative string|nil Preferred alternative function.
+---@param version     string     Neovim version in which the deprecated
+---                              function will be removed.
+function vim.deprecate(name, alternative, version)
+    local message = name .. ' is deprecated'
+    message = alternative and (message..', use '..alternative..' instead.') or message
```

Comment:
not sure if you actually care about this or if it's just something I do
but in this line there is no space around `..` everywhere else there is

Reply:
Oh, you're right. Thanks.